apps/src/JsDebuggerUi.js

```diff
@@ -31,9 +32,10 @@ var WATCH_VALUE_NOT_RUNNING = "undefined";
  * App Lab, Game Lab, etc.
  * @param {!function} runApp - callback for "launching" the app, which is used
  *        by the "Step In" button when the app isn't running.
+ * @param {!ReduxStore} reduxStore
```

Comment:
Update: Anyplace we used this `ReduxStore` type should just be `Store` (it's what [`createStore` is documented to return](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/redux.js#L17)).